src/nimble.nim

```diff
@@ -858,6 +860,19 @@ js   - Compile using JavaScript backend.""",
     pkgRoot
   )
 
+  # Add git vcs to the new project
+  let repo = createGitRepo(path=pkgRoot)
+  discard repo.gitCommand("init")
+
+  var fd = open(joinPath(pkgRoot, ".gitignore"), fmWrite)
```

Comment:
library packages dont produce executables except for tests. and nothing ever produces `*.out` unless you explicitly do that.